src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * The maximum bitrate available for this guild
+   * @type {number}
+   * @readonly
+   */
+  get maximumBitrate() {
+    if (this.features.includes('VIP_REGIONS')) {
+      return 384000;
+    } else {
+      switch (this.premiumTier) {
+        case PremiumTiers[this.premiumTier] === 0:
+          return 96000;
+        case PremiumTiers[this.premiumTier] === 1:
+          return 128000;
+        case PremiumTiers[this.premiumTier] === 2:
+          return 256000;
+        case PremiumTiers[this.premiumTier] === 3:
+          return 384000;
+        default:
+          return 96000;
+      }
```

Comment:
this isn't how switch statements work...

Reply:
> this isn't how switch statements work...

You are right. I did it all wrong. I think that the last commit would be the best.


>Alternatively, we can also define an object somewhere and do the following:

The reason i went with a switch statement is that if tomorrow, discord releases a Tier 4, we won't have an issue as it would at least return the default bitrate.  But other options would also work.